doc/book/cookbook/debug-toolbars.md

```diff
@@ -74,11 +74,9 @@ return [
         ],
     ],
     'middleware_pipeline' => [
-        'pre_routing' => [
-            [
-                'middleware' => [
-                    PhpDebugBarMiddleware::class,
-                ],
+        [
+            'middleware' => [
+                PhpDebugBarMiddleware::class,
             ],
```

Comment:
Should have routing and dispatch middlewares here to illustrate that `PhpDebugBarMiddleware::class` must be piped before them?